Scripts/Python/nb01RPSGame.py

```diff
@@ -389,6 +394,10 @@ def OnNotify(self, state, id, events):
 
         # Finished sitting down.
         if self._HandleNotify(state, id, events, self._sitting, self._OnSitDown):
+            if not self.playing:
+                PtEnableControlKeyEvents(self.key)
+                PtDisableMovementKeys()
+                PtEnableMouseMovement()
```

Comment:
This should probably be done in `nb01RPSGame._OnSitDown()`. I wanted the `OnNotify()` method to be as slim as possible with effectively no gameplay or interface logic.